menus/significant-other.cson

```diff
@@ -4,7 +4,8 @@
     'submenu': [
       'label': 'Significant Other'
       'submenu': [
-        { 'label': 'Toggle', 'command': 'significant-other:toggle' }
+        { 'label': 'Toggle', 'command': 'significant-other:toggle' },
+        { 'label': 'Clear Cache', 'command': 'significant-other:clear-cache' }
```

Comment:
Do we need to provide a command for clearing the cache?

As long as we verify that the cached file path exists on disk before attempting to open that path, I don't _think_ users will need to manually clear the cache. What do you think?